src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
+    }
+
+    public Ball(String color, int number) {
+        this.ballColor = color;
+        this.ballNumber = number;
+    }
+
+    public String getBallColor() {
+        return this.ballColor;
+    }
+
+    public void setBallColor(String ballColor) {
+        this.ballColor = ballColor;
+    }
+
+    public int getBallNumber() {
+        return this.ballNumber;
+    }
+
+    public void setBallNumber(int ballNumber) {
+        this.ballNumber = ballNumber;
+    }
+
+    public String toString() {
+        String ball = "ball";
+        return "Ball with number: " + ballNumber + " and color: " + ballColor;
```

Comment:
Please use `return String.format("Ball with number: %d and color: %s", this.ballNumber, this.ballColor);` or
`return "Ball with number: %d and color: %s".formatted(this.ballNumber, this.ballColor);`

> Direct string concatination should be avoided. You can check all the format specifiers via this [link](https://www.javatpoint.com/java-string-format#:~:text=Java%20String%20Format%20Specifiers)